### PR TITLE
Create object IO sample

### PR DESCRIPTION
**Pull request by flei2000**

### CODE REVIEW

File: ecstest/testcases/dataplane/object_io.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    def tearDown(self):
+        self.data_conn.delete_bucket(self.bucket_name)
+        super(TestObjectPostGetDelete, self).tearDown()
+
+    def test_basic_post_get_delete(self):
+        """JIRA ID: ECSTEST-5"""
+        k = Key(self.bucket)
+        k.key = 'test_key_1'
+        content = 'This the the content of test_key_1'
```
Comment (stevendgonzales) [inline ecstest/testcases/dataplane/object_io.py]: i would name this var "expected_content"

File: ecstest/testcases/dataplane/object_io.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        k.set_contents_from_string(content)
+
+        content_2 = k.get_contents_as_string()
```
Comment (stevendgonzales) [inline ecstest/testcases/dataplane/object_io.py]: i would then name this var "actual_content"

File: ecstest/testcases/dataplane/object_io.py (hunk 1 above)
Comment (MingatRubicon) [inline ecstest/testcases/dataplane/object_io.py]: consider to add a reusable class for acquire content. sub class could be provide fixed string. content from /dev/urandom, file.

Comment (flei2000) [inline ecstest/testcases/dataplane/object_io.py]: Let's implement this function in another case with 1B to 40MB content.

File: ecstest/testcases/dataplane/object_io.py (hunk 2 above)
Comment (flei2000) [inline ecstest/testcases/dataplane/object_io.py]: Personally, these two names are still confusing. I'd like to rename content_2 to content_ret.

**Comment by stevendgonzales** [comment]
closing, captured in PR #10

**Pull request closed without merging**